app/Http/Controllers/SquareCheckoutController.php

```diff
@@ -95,6 +95,10 @@ public function payDues(Request $request)
 
                 $amount = (int) ($transaction->package->cost * 100);
 
+                if ($payment->url !== null) {
+                    return redirect($payment->url);
```

Comment:
Is any validation needed on this? Injection risk or similar

Reply:
No, it originates from Square (in the `redirect` method)